index.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const { Client, Collection, Intents } = require('discord.js');
+const { token } = require('./config.json');
+const Keyv = require('keyv');
+
+const userPrefs = new Keyv('sqlite://users.db');
+userPrefs.on('error', err => console.error('Keyv connection error:', err));
+
+const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.DIRECT_MESSAGES] });
+
+
+client.commands = new Collection();
+const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
const commandFiles = fs.readdirSync('./commands').filter((file) => file.endsWith('.js'));
```